pkg/diagnostics/network/objects.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	diagnosticsImage           = "openshift/diagnostics-deployer"
+	diagnosticsImage           = "openshift/origin"
```

Comment:
Is it appropriate to use the image format string and strip the hyphen? That's the image name we'd want 'openshift/origin' and 'openshift3/ose'